stratum/hal/lib/common/p4_service.cc

```diff
@@ -303,6 +304,12 @@ ::grpc::Status P4Service::Write(::grpc::ServerContext* context,
                           "Write from non-master is not permitted.");
   }
 
+  // Check bytestrings here.
+  ::util::Status ret = IsInCanonicalByteStringFormat(*req);
+  if (!ret.ok()) {
+    return ToGrpcStatus(ret, {});
+  }
+
```

Comment:
According to spec, we must accept strings in all formats. Don't think we need this check.